testutils/build.gradle

```diff
@@ -54,8 +94,23 @@ dependencies {
     implementation "com.google.code.gson:gson:$rootProject.ext.gsonVersion"
     implementation "com.nimbusds:nimbus-jose-jwt:$rootProject.ext.nimbusVersion"
     implementation 'androidx.appcompat:appcompat:1.0.2'
-    testImplementation 'junit:junit:4.12'
+    implementation "junit:junit:$rootProject.ext.junitVersion"
+    implementation "com.opencsv:opencsv:4.1"
```

Comment:
OSS component reviewed?

Reply:
Nope, Thanks for pointing out. Will do that. For test only dependencies, that actually just means registering the component in Component Governance.